rq/cli/cli.py

```diff
@@ -267,7 +267,8 @@ def worker(
             queues,
             name=name,
             connection=cli_config.connection,
-            default_worker_ttl=worker_ttl,
+            default_worker_ttl=worker_ttl,  # TODO remove this arg in 2.0
```

Comment:
Mind raising a `DeprecationWarning` if `default_worker_ttl` is passed?

Reply:
One problem is that the user can supply their own `worker_class`, so until there is a breaking 2.0 change which removes that arg I don't think we reliably tell if `default_worker_ttl` was passed in.

However If the built in worker class is used this is what the log looks like:

```
venv ❯ python -W default -m rq.cli worker default
/Users/sean/repos/rq/rq/cli/cli.py:266: DeprecationWarning: default_worker_ttl is deprecated. Use worker_ttl instead.
  worker = cli_config.worker_class(
10:05:59 Worker rq:worker:ebde86e466874bc0b60e6222354243ec started with PID 51139, version 1.16.2
```

Notice it is pointing to line 266 which is where we instantiate the class

`cli/cli.py:266: DeprecationWarning: default_worker_ttl is deprecated. Use worker_ttl instead.`

So we indicate to the user that this argument is being deprecated at the usage anyways. This is the reason why I added `stacklevel=2` in the deprecation warning as it logs where the usage occurs rather than the line where the deprecation warning is.

Let me know if that makes sense or if you would like some other adjustments.